<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="4">
     <widget class="QPushButton" name="ButtonF">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QPushButton" name="ButtonXOR">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>XOR</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="Button2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="ButtonSub">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #FF9500;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #EE7B00;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="ButtonRightShift">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QPushButton" name="ButtonOR">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>OR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="ButtonOctet">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #656565;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="Button4">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="ButtonC">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QPushButton" name="ButtonAdd">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #FF9500;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #EE7B00;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="Button9">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5" colspan="2">
     <widget class="QPushButton" name="ButtonEquals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #FF9500;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #EE7B00;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="ButtonE">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="Button8">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Button7">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="ButtonMul">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #FF9500;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #EE7B00;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="Button1">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
 background-color: #505050;
 color: #FFFFFF;
 border: 1px solid grey;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="ButtonDecimal">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #656565;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="ButtonNOR">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>NOR</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="ButtonA">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="Button5">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QPushButton" name="Button00">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="ButtonX">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>X&gt;&gt;Y</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="Button3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="ButtonB">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="ButtonHexa">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #656565;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="ButtonDiv">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #FF9500;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #EE7B00;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="ButtonLeftShift">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="ButtonD">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="Button0">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="ButtonY">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>X&lt;&lt;Y</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="Button6">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="ButtonFF">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #D4D4D2;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>FF</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="ButtonAND">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>AND</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QPushButton" name="ButtonClear">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: #505050;
  border: 1px solid gray;
  padding: 5px;
  color: #FFFFFF;
}
QPushButton:pressed {
     background-color: #A9A9A9;
    border: 1px solid gray;
    padding: 5px;
    color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
